src/main/java/no/nav/tag/finnkandidatapi/tilgangskontroll/TilgangskontrollService.java

```diff
@@ -0,0 +1,45 @@
+package no.nav.tag.finnkandidatapi.tilgangskontroll;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.tag.finnkandidatapi.kandidat.Kandidat;
+import no.nav.tag.finnkandidatapi.kandidat.Veileder;
+import no.nav.tag.finnkandidatapi.tilgangskontroll.veilarbabac.VeilarbabacClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.util.function.Supplier;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class TilgangskontrollService {
+    private final VeilarbabacClient veilarbabacClient;
+    private final TokenUtils tokenUtils;
+
+    public boolean harSkrivetilgangTilKandidat(String fnr) {
```

Comment:
Hva er forskjell på denne metoden og `sjekkSkrivetilgangTilKandidat`? Er det meningen at vi skal ha begge to?